### ISSUE TITLE
bug

### ISSUE THREAD

**Issue by foldar** [git-for-windows/git#3136]
I am used to microsoft visual sourcesafe and Sourcegear Vault, the whole git is a bug to me.

I suppose it can do version control of software, and check in and check out files, 
but on a first look it doesnt compare to those two. 
it doesn't even have a graphical interface.

Lots to improve. Can it do difference checks?

The help file is not clear it doesn't even have the commands checkin and checkout.
Am I supposed to work with this?

Bad software.

With kind regards,

Nico Felten

**Comment by foldar** [comment]
That was the shell program i started up, what good is a dos prompt when i want to start up git?

**Comment by foldar** [comment]
No drag drop files into git, bad.

**Comment by foldar** [comment]
on drag drop a file into explorer, the git screen doesnt refresh with the new file unless i tell it to

**Comment by foldar** [comment]
I'ld say have a look at sourcegear vault for good software for checking in and out files. And make it more like that. For that software isnt free.

**Comment by rimrul** [comment]
> I am used to microsoft visual sourcesafe and Sourcegear Vault, the whole git is a bug to me.

Feel free to use those then.

> it doesn't even have a graphical interface.

Git for Windows comes with Git GUI and Gitk. There are also lots of third party clients that act as a Graphical interface for git

> The help file is not clear

Feel free to suggest improvements on the git mailing list.

> it doesn't even have the commands checkin and checkout.

We do have [`git checkout`](https://git-scm.com/docs/git-checkout). You're right about the fact there isn't a `git checkin`. Maybe [`git add`](https://git-scm.com/docs/git-add) and [`git commit`](https://git-scm.com/docs/git-commit) are what you're looking for?

> Am I supposed to work with this?

I'm not forcing you.

> what good is a dos prompt when i want to start up git?

A Terminal for using a CLI based application? Pretty good.

**Comment by foldar** [comment]
I think visual sourcesafe is no longer available with higher visual studio versions.
And sourcegear vault is 350-450 $ per license.

The company I applied with and studied git for is useing git (free of cost), so I wouldnt really have a choice,
except letting them know there is something better but expensive.

For suggesting improvements, there are too many to mention,
I would suggest to have a look at sourcegear vault and make it more like that.

There is a graphical interface, I mistook the dos prompt that started after install for git and was unsatisfied.
It should start up git instead, not a dos prompt.

Also when I looked for git at the internet i started with git hub and didnt see how
i could add a file to github, or check in things in there. Only later i found git itself.
Avoid the confusion between the two and better describe which does what.
It should be clear to an expert like me. Maybe I should ask a friend to teach me git a bit before commenting more.

It was justa  first impression.
Though I am sure git is to source gear vault like notepad is to word.

With kind regards,

Nico Felten
________________________________
